### DISCUSSION TITLE
I would like to publish lxp/BAxxxxx/inputs/all

### DISCUSSION THREAD

**Discussion by 6spiderman** [celsworth/lxp-bridge#160] [Q&A]
Good day

I am quite new to MQQT, I played around a bit and used MQTT Explored to view all packets published. Is there a way to publish lxp/BAxxxxxx/inputs/all as I would Like to add v_ac_r to setup automations in HA.

**Comment by celsworth** [comment]
Hi :)

`lxp/BAxxxxxx/inputs/all` is published at the same as `inputs/3`. And this is published when lxp-bridge receives the final set of input registers from the inverter.

A way to request this being published on demand or more regularly might be a work in progress for one of the next versions, but for now you can request all the inputs (including `all`) is to script something like this:

```
mosquitto_pub -t lxp/cmd/all/read/inputs/1 -m ''
sleep 2
mosquitto_pub -t lxp/cmd/all/read/inputs/2 -m ''
sleep 2
mosquitto_pub -t lxp/cmd/all/read/inputs/3 -m ''
```

So basically request each set of inputs registers in turn, with a small delay to let things catch up. After publishing the final cmd you should see `inputs/all` appear in MQTT.

As I say its a bit messy for now, hopefully it gets improved in a future version, time and motivation permitting ;)

**Comment by 6spiderman** [comment]
Hi Chris

Wow! Thank you for the quick response.

I will give this a try! Really appreciate it.

Kind Regards
Christopher
